core/store/src/migrations.rs

```diff
@@ -707,6 +707,15 @@ pub fn migrate_21_to_22(path: &String) {
     set_store_version(&store, 22);
 }
 
+pub fn migrate_25_to_26(path: &String) {
+    let store = create_store(path);
+    let mut store_update = store.store_update();
+    store_update.delete_all(DBCol::ColCachedContractCode);
+    store_update.commit().unwrap();
+
+    set_store_version(&store, 26);
+}
```

Comment:
I think it's fine to just completely bust the precompilation cache at this time, as we haven't yet deployed compilation on contract deployment to the main net, and the code is written in such a way that empty cache causes slowdowns, rather then errors, so testnet should be fine. 

That is, it's fine to land this without tackling https://github.com/near/nearcore/issues/4473 first. @bowenwang1996 please veto this if I am mistaken in my assumptions here.